dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -205,6 +206,74 @@ private Workflow manualRegisterAndPublish(WorkflowsApi workflowsApi, String work
         return workflow;
     }
 
+    // Tests 6 things:
+    // WDL workflow with local imports
+    // WDL workflow with HTTP imports
+    // CWL workflow with local imports
+    // CWL workflow with HTTP imports
+    // WDL workflow with HTTP imports and local imports and nested
+    // CWL workflow with HTTP imports and local imports and nested
+    @Test
+    public void testLanguageParsingInformation() {
```

Comment:
Can this be split into two shorter tests for CWL and WDL separately?
(Makes it easier to debug in the future if one language parsing method is broken)